js/map-content-to-topper.js

```diff
@@ -0,0 +1,162 @@
+const themeImageRatio = {
+	'split-text-center': 'split',
+	'split-text-left': 'split',
+	'full-bleed-image-center': 'full-bleed',
+	'full-bleed-image-left': 'full-bleed',
+	'full-bleed-offset': 'full-bleed'
+};
+
+const isNews = content => content.annotations && content.annotations.find(a => a.prefLabel === 'News');
+
+const getTopperSettings = content => {
+	content.topper = content.topper || {};
+	//live blogs and news packages
+	if (content.realtime && content.liveBlog ||
+		content.package && isNews(content.package) && content.package.contains[0].id === content.id ||
+		content.type === 'package' && isNews(content)) {
+
+		const designTheme = content.package && content.package.design.theme || content.design && content.design.theme;
```

Comment:
There will be logic to do with theme/design that's not included here. I think it's still in article and due to be moved to ES eventually although it's not entirely possible to do it for content packages and Special Reports yet. The theme can get overridden based on metadata.

Reply:
as in, you think there's more logic in `next-article` i've not accounted for? i didn't find anything

Reply:
We'll that's understandable given the baroque nature of the article app 😄 . For example: https://github.com/Financial-Times/next-article/blob/master/server/onward-journey/decorate-content.js#L8

Reply:
And https://github.com/Financial-Times/next-article/blob/master/server/model/helpers/design.js

Reply:
`HACK: temporary, remove after packaging refactor (LK)`
😅